frontstage/common/eq_payload.py

```diff
@@ -46,7 +46,7 @@ def create_payload(self, case, party_id, business_party_id, survey):
         collex_event_dates = self._get_collex_event_dates(collex_id)
 
         # Party
-        party = party_controller.get_party_by_business_id(business_party_id, collection_exercise_id=collex_id)
+        party = party_controller.get_party_by_business_id(current_app.config, business_party_id, collection_exercise_id=collex_id)
```

Comment:
One for the team , but should we pass the entire config or just the values we need ?

Reply:
Fewer parameters = better

Reply:
I spoke to Torrance and John about it, because we're only passing in two config values we agreed to just to pass them in instead of the entire object